README.md

```diff
@@ -1 +1,27 @@
-# ancestry
\ No newline at end of file
+# ancestry
+
+Prioritising and assigning ancestry to populations.
+
+## Running hail
+
+To connect  to hail with the required gnomAD dependencies, run the following command:
+
+```hailctl dataproc start --max-age 2h --region australia-southeast1 my-cluster --packages gnomad```
```

Comment:
If you want to split the command up a bit for more clarity, you can use `\` at the end of each line - bash will treat the next line as a continuation of the command:

```
hailctl dataproc start \
  --max-age 2h \
  --region australia-southeast1 \
  --packages gnomad \
  my-cluster 
```